google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/DefaultCloudSdkService.java

```diff
@@ -84,9 +85,10 @@ public static CloudSdkVersion getMinimumRequiredCloudSdkVersion() {
   @Nullable
   @Override
   public Path getSdkHomePath() {
-    // For Windows users that persisted the old malformed path.
-    if (validateCloudSdk(propertiesComponent.getValue(CLOUD_SDK_PROPERTY_KEY))
-        .contains(CloudSdkValidationResult.MALFORMED_PATH)) {
+    // To let Windows users that persisted the old malformed path save a new one.
+    // TODO(joaomartins): Delete this after a while so gets are faster.
+    if (isMalformedCloudSdkPath(propertiesComponent.getValue(CLOUD_SDK_PROPERTY_KEY))) {
+      UsageTrackerProvider.getInstance().trackEvent("malformedPath").ping();
```

Comment:
Sorry @chanseokoh , I was too fast to trigger the merge. I'll submit a new PR doing so, although the plan is to remove this in January or something. Do you think it's still worth adding it there?